lib/browser/browser.ts

```diff
@@ -430,6 +431,17 @@ class OktaAuthBrowser extends OktaAuthBase implements OktaAuth, SignoutAPI {
     const storage = browserStorage.getSessionStorage();
     storage.removeItem(REFERRER_PATH_STORAGE_KEY);
   }
+
+  isAuthorizationCodeFlow(): boolean {
+    const responseType = this.options.responseType || '';
+    let isCodeResponse = false;
+    if (Array.isArray(responseType) && responseType.length) {
+      isCodeResponse = responseType.indexOf('code') >= 0;
+    } else {
+      isCodeResponse = responseType === 'code';
+    }
+    return (!this.options.pkce && isCodeResponse);
```

Comment:
I am probably wrong, my understanding is authorization code flow is a super set of pkce flow.

Reply:
you are probably right. we need a way of knowing whether we expect to redirect or not. For us this is web/native apps with "authorization_code" flow who are not using PKCE. Open to suggestions for better naming here.

Reply:
A proper name can be complex, how about 
```
isAuthorizationCodeFlow(pkce?: boolean): boolean 
```

Reply:
I don't follow the impact of the passed boolean.

If this is checking for an Authorization code flow without PKCE, I'd say 
`isAuthorizationCodeFlowWithoutPKCE()` - a little Java-esque, but not confusing

If want to cover both, I'd be more explicit in the param:
`isAuthorizationCodeFlow({ withPKCE?: boolean }) `
- Ideally `withPKCE` would be required
- using the object so that it's clear, as (`isAuthorizationCodeFlow(true)` is not clear)
- This still makes it less than perfectly clear what happens if you pass `false`, which is why I prefer the explicit name above.

Reply:
@shuowu I don't want to accept arguments, because the point of this method is for AuthJS to tell someone else whether it is operating in (non-PKCE) authorization code flow. Maybe if I had another function `isPKCE` then the caller can say `!isPkce() and isAuthorizationCodeFlow()` to get the same meaning.

Reply:
that would make sense! our pkce flow actually is pkce + ahthorization code flow,that's where the confusion is introduced, imo

Reply:
`!isPkce() and isAuthorizationCodeFlow()` - this looks ideal to me.